apps/system/js/app_window_manager.js

```diff
@@ -730,6 +731,7 @@
     },
 
     '_handle_sheets-gesture-begin': function() {
+      this.sheetTransitioning = true;
```

Comment:
nit: `this._sheetTransitioning` and "declare" it with a comment like the `_activeApp` private var.